### DISCUSSION TITLE
Ability to hide a specific endpoint

### DISCUSSION THREAD

**Discussion by rbzinnia** [parvez3019/go-swagger3#50] [General]
Do we have any annotation to hide a specific endpoint from being generated in the doc? If not could we please get that added?

**Comment by parvez3019** [comment]
don't add comments for that endpoint it will not be added

**Comment by rbzinnia** [comment]
Umm that would be hard to maintain right - removing the comments - then generating oas.json - then adding comments back and checking in. Is it possible to build some kind of annotation or command line flag which ignores certain comments?

**Reply by parvez3019** [reply]
Go lang doesn't support annotations that's why this library and all other libraries for this job works with comments 

Comments or annotations both require equal amount of work in maintenance 

In springboot we have controller annotations from where it is easier to find out api methods but with go lang frameworks we don't have any of that.

**Reply by rbzinnia** [reply]
okay thanks for the explanation.
